tools/openshift-ci/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM quay.io/centos/centos:7
+
+LABEL vendor="Red Hat inc."
+LABEL maintainer="AOS QE Team"
```

Comment:
perhaps we should change it to `OCP QE Team` instead... `AOS` is a legacy name that's not commonly used anymore.

Reply:
@pruan-rht Fix that in https://github.com/openshift/verification-tests/pull/2154